boards/riscv/hifive1/hifive1.dts

```diff
@@ -25,16 +29,32 @@
 	};
 
 	leds {
-		compatible = "pwm-leds";
+		compatible = "gpio-leds";
 		led0: led_0 {
-			pwms = <&pwm1 1 PWM_MSEC(20)>;
+			gpios = <&gpio0 19 GPIO_ACTIVE_LOW>;
 			label = "Green LED";
 		};
 		led1: led_1 {
-			pwms = <&pwm1 2 PWM_MSEC(20)>;
+			gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
 			label = "Blue LED";
 		};
 		led2: led_2 {
+			gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+			label = "Red LED";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwmled0: pwmled_0 {
+			pwms = <&pwm1 1 PWM_MSEC(20)>;
```

Comment:
Should it be `pwms = <&pwm1 1 PWM_MSEC(20) PWM_POLARITY_INVERTED>;`?

Reply:
Currently hifive1's PWM only takes channel and period parameters. I'll leave it as is for this PR.